op-node/flags/flags.go

```diff
@@ -41,7 +41,7 @@ var (
 	DaRPC = cli.StringFlag{
 		Name:   "da-rpc",
 		Usage:  "Data Availability RPC",
-		Value:  "http://da:26659",
+		Value:  "http://localhost:26659",
```

Comment:
This probably needs to get changed for it to work correctly right?

Reply:
good catch! fixed here https://github.com/celestiaorg/optimism/pull/32/commits/aff0750dafb557042e74f8b367f991d026d938be